<ui version="4.0" >
 <class>Config</class>
 <widget class="QWidget" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Information Entities</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="toolTip" >
           <string>Maximum size of the vector.</string>
          </property>
          <property name="whatsThis" >
           <string>Specify the maximum number of non-zero weighted information entities used for the description of the subprofile.</string>
          </property>
          <property name="text" >
           <string>Maximum Size</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KIntNumInput" name="MaxSize" >
          <property name="toolTip" >
           <string>Maximum size of the vector.</string>
          </property>
          <property name="whatsThis" >
           <string>Specify the maximum number of non-zero weighted information entities used for the description of the subprofile.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2" >
        <item>
         <widget class="QLabel" name="textLabel1bis" >
          <property name="toolTip" >
           <string>Number of irrelevant entities.</string>
          </property>
          <property name="whatsThis" >
           <string>Specify the number of irrelvant weighted information entities used for the description of the subprofile.</string>
          </property>
          <property name="text" >
           <string>Number of irrelevant entities</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2bis" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KIntNumInput" name="NegSize" >
          <property name="toolTip" >
           <string>Number of irrelevant entities.</string>
          </property>
          <property name="whatsThis" >
           <string>Specify the number of irrelvant weighted information entities used for the description of the subprofile.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="IncrementalMode" >
        <property name="toolTip" >
         <string>Specify if the incremental mode must be used.</string>
        </property>
        <property name="whatsThis" >
         <string>When this parameter is checked, the method computes the description with the old description as a "virtual document" and the "new" documents.</string>
        </property>
        <property name="text" >
         <string>Incremental mode used</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Factors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="_3" >
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="toolTip" >
           <string>Specify the relevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the relevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="text" >
           <string>Relevant Factor</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KDoubleNumInput" name="RelFactor" >
          <property name="toolTip" >
           <string>Specify the relevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the relevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="minValue" stdset="0" >
           <number>-1</number>
          </property>
          <property name="maxValue" stdset="0" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_4" >
        <item>
         <widget class="QLabel" name="textLabel4" >
          <property name="toolTip" >
           <string>Specify the fuzzy relevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the fuzzy relevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="text" >
           <string>Fuzzy Factor</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>28</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KDoubleNumInput" name="FuzzyFactor" >
          <property name="toolTip" >
           <string>Specify the fuzzy relevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the fuzzy relevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="minValue" stdset="0" >
           <number>-1</number>
          </property>
          <property name="maxValue" stdset="0" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_5" >
        <item>
         <widget class="QLabel" name="textLabel5" >
          <property name="toolTip" >
           <string>Specify the irrelevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the irrelevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="text" >
           <string>Irrelevant Factor</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KDoubleNumInput" name="IrrelFactor" >
          <property name="toolTip" >
           <string>Specify the irrelevant factor used.</string>
          </property>
          <property name="whatsThis" >
           <string>During the computation, all the irrelevant documents assessed by the profile have a specific weight corresponding to this parameter.</string>
          </property>
          <property name="minValue" stdset="0" >
           <number>-1</number>
          </property>
          <property name="maxValue" stdset="0" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
